api/src/services/pokemonSyncronizer.service.ts

```diff
@@ -1,8 +1,9 @@
+import { SpanKind } from '@opentelemetry/api';
 import { getPokemonRepository, Pokemon } from '@pokemon/repositories';
 import { createQueueService } from '@pokemon/services/queue.service';
 import { createSpan, getParentSpan, runWithSpan } from '@pokemon/telemetry/tracing';
 
-export const MESSAGE_GROUP = '/queue/syncronizePokemon';
+export const MESSAGE_GROUP = 'queue.synchronizePokemon';
```

Comment:
Changing this `MESSAGE_GROUP` to follow the OTEL convention (with a dot), not sure if this could affect the implementation. Is it required to have `/xxx/xxxx` for this queue?